src/components/select/select.template.tsx

```diff
@@ -32,10 +32,12 @@ export const SelectTemplate: FunctionalComponent<SelectTemplateProps> = (
 ) => {
     const value = props.value;
     let hasValue = !!props.value;
+    let hasEmptyText = true;
```

Comment:
`hasEmptyText` sounds a bit confusing, what if we say `hasEmptyString` or `hasNoText` 🤔

Reply:
Good point. The reason is that items in the Select have both `value` and `text`.
<img width="438" alt="Screenshot 2022-11-30 at 07 29 30" src="https://user-images.githubusercontent.com/35954987/204724130-864ad526-d011-4895-b5e0-76330c3daa6f.png">
Even empty items which we wanna hide have `value` (some ID in the database in case of CRM). But their `text` is empty.

Why we say "empty" and for example not "hasNoText"?
1. it's inspired by the [`:empty` pseudo selector](https://developer.mozilla.org/en-US/docs/Web/CSS/:empty).
2. It has a "text" string, but the string is empty.

Reply:
I see, I think this confusion was caused by the similarity of `value` and `text`, would be better to have `id` instead of `value`. 

but it's another topic. 
I would vote to have `hasNoText` but it's up to you.